docs/local-setup.md

```diff
@@ -94,7 +94,6 @@ php artisan queue:restart
 php artisan schedule:interrupt
 npm ci
 php artisan filament:upgrade
-php artisan app:build-assets
```

Comment:
Should this be replaced with `npm run build`? Or is the thought that `npm run build` will have been run in the container by s6?

Reply:
Nope, I think that's right just wanted to confirm.